src/store/modules/actions/wallet.js

```diff
@@ -79,3 +81,12 @@ export async function initIfNecessary({ dispatch }) {
     await dispatch('initWallet', { accounts, offlineSigner, method });
   }
 }
+
+export async function walletFetchLIKEBalance({ commit }, address) {
```

Comment:
This action doesn't need `address` as param when we can get it from the state.

```suggestion
export async function walletFetchLIKEBalance({ commit, state }) {
  const { address } = state;
```